source/_cookbook/dim_and_brighten_lights.markdown

```diff
@@ -141,11 +141,11 @@ script:
           data_template:
             entity_id: light.YOURLIGHT
             brightness: >
```

Comment:
Please add a hyphen/minus (`-`) after the `>` here to tell YAML to trim leading and trailing whitespace.